frontend/packages/shared/src/components/ToggleFilterGroup/ToggleFilterGroup.tsx

```diff
@@ -33,7 +33,7 @@ export function ToggleFilterGroup<T>({
         >
           <div className="columns gapped">
             <div className="grow">{label}</div>
-            {count && <div>{`(${count})`}</div>}
+            {typeof count === "number" && <div>{`(${count})`}</div>}
```

Comment:
```suggestion
            {typeof count === "number" && !isNaN(count) && <div>{`(${count})`}</div>}
```

Jos laittas vielä varuiksi `!isNaN()`?